main.py

```diff
@@ -2,6 +2,27 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+from db.data import GENRES, ACTORS
+
 
 def main() -> QuerySet:
-    pass
+
+    for genre in GENRES:
+        Genre.objects.create(name=genre)
+
+    for actor in ACTORS:
+        first_name, last_name = actor
```

Comment:
u can unpack actor variable in loop inicialization 
like: `first_name, last_name in Acotrs`